src/unlock/omni_lock.rs

```diff
@@ -101,10 +129,12 @@ bitflags! {
 /// 4. 2 bytes minimum ckb/udt in ACP, optional
 /// 5. 8 bytes since for time lock, optional
 /// 6. 32 bytes type script hash for supply, optional
-#[derive(Clone, Serialize, Deserialize, Debug, Hash, Eq, PartialEq)]
+#[derive(Clone, Serialize, Deserialize, Hash, Eq, PartialEq)]
 pub struct OmniLockConfig {
     /// The auth id of the OmniLock
     pub id: Identity,
+    /// The multisig config.
+    pub multisig_config: Option<MultisigConfig>,
     /// The omni lock flags, it indicates whether the other four fields exist.
     pub omni_lock_flags: OmniLockFlags,
 }
```

Comment:
OmniLockConfig's fields should not be public since user may construct a bad data cause panic. For example:
```
let cfg = OmniLockConfig {
    id: Identity::from(...),
    multisig_config: None,
    omni_lock_flags: OmniLockFlags::Multisig,
}
```
This will let most operation panic, since the sdk code will call `cfg.multisig_config.unwrap()`